assets/mails/notifications_planchanged.mjml

```diff
@@ -0,0 +1,18 @@
+{{define "content"}}
```

Comment:
[nit] I would have put an underscore in the filename between `plan` and `changed`

Reply:
I decided to copy the existing pattern for notification e-mails templates but I agree with you.
Let's change that.